docs/design/w3c/spec.md

```diff
@@ -0,0 +1,545 @@
+## W3C Verifiable Credentials Representation
+
+This section describes how Indy-styled AnonCreds credentials can be represented in the form of W3C Verifiable
```

Comment:
Throughout this document, the term “Indy” should be removed, as AnonCreds are stand-alone and independent of Indy.  In most cases in this text, “AnonCreds” is suitable replacement. Obviously not in this case 😄.  I think we should use the terms “legacy AnonCreds format” and “W3C AnonCreds format” for the type representations. Where necessary “VC” and “VP” would be put in front of “format”.